modular_skyrat/modules/primitive_cooking_additions/code/millstone.dm

```diff
@@ -46,6 +57,10 @@
 	drop_everything_contained()
 	balloon_alert(user, "removed all items")
 
+/obj/structure/millstone/CtrlShiftClick(mob/user)
+	set_anchored(!anchored)
+	balloon_alert(user, "[anchored ? "secured" : "unsecured"]")
```

Comment:
Why does it get unanchored with ctrl-shift-click rather than using a wrench? That's kinda inconsistent with just about everything else in the game.

Reply:
Because we can't expect primitives to have tools with them at any time ever. I'd rather they be able to move it around with a simple click action rather than force every primitive role to have a full set of tools to do anything.

Reply:
Hmmm, okay.